pkg/serverless/invocationlifecycle/trace.go

```diff
@@ -81,6 +85,11 @@ func endExecutionSpan(processTrace func(p *api.Payload), requestID string, endTi
 		},
 	}
 
+	if strings.ToLower(os.Getenv("DD_CAPTURE_LAMBDA_PAYLOAD")) == "true" {
```

Comment:
could you add this env variable as a const on top of the file?

Reply:
added